.github/workflows/translations-sync-dw.yml

```diff
@@ -0,0 +1,35 @@
+name: Sync translations in desktop wallet
+
+on:
+  workflow_dispatch:
+  push:
+    paths: ['apps/desktop-wallet/locales/**']
+    branches: [next]
```

Comment:
This action will run only if there are changes in the files in the defined `paths` when a push in the `next` branch happens, or when run manually (`workflow_dispatch`).